### PR TITLE
Release v0.2.0 (Bernina Express)

### PR DESCRIPTION
**Pull request by iaguis**
:steam_locomotive:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,286 @@
...
+component "prometheus-operator" {
+    namespace = "<NAMESPACE>"
+    grafana {
+        admin_password = "<GRAFANA_PASSWORD>"
+    }
+    # etcd_endpoints is not needed anymore
+}
+```
+
+Check out the new syntax in the [Prometheus Operator configuration reference](https://github.com/kinvolk/lokomotive/blob/v0.2.0/docs/configuration-reference/components/prometheus-operator.md) for details.
```
Comment (knrt10) [inline CHANGELOG.md:141]: This currently gives 404, will it be available after release?

Comment (iaguis) [reply CHANGELOG.md:141]: Yes, after this PR is merged I will tag v0.2.0 and the link should work.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,286 @@
...
+#### UX
+
+- Add `--confirm` flag to delete component without asking for confirmation ([#568](https://github.com/kinvolk/lokomotive/pull/568)).
+- Add error message for missing ipxe_script_url ([#540](https://github.com/kinvolk/lokomotive/pull/540)).
+- Show logs when terraform fails in `lokoctl cluster apply/destroy` ([#323](https://github.com/kinvolk/lokomotive/pull/323)).
+- cli/cmd: rename --kubeconfig flag to --kubeconfig-file ([#602](https://github.com/kinvolk/lokomotive/pull/602)). This is because cobra/viper consider the KUBECONFIG environment variable and the --kubeconfig flag the same and this can cause surprising behavior.
+
+#### Docs
+
+- docs: make quickstart quick (TODO)
```
Comment (knrt10) [inline CHANGELOG.md:253]: Should there be any link?

Comment (iaguis) [reply CHANGELOG.md:253]: Right, thanks for the catch!

**Comment by ipochi** [comment]
I assume that docs link pointing to `v0.2.0` will be resolved once the `v0.2.0` tag is created ?

```
markdown-link-check CHANGELOG.md

FILE: CHANGELOG.md

[✓] #component-updates
[✓] #security
[✓] #changes-in-v020
[✖] https://github.com/kinvolk/lokomotive/blob/v0.2.0/docs/configuration-reference/platforms/packet.md#attribute-reference
[✖] https://github.com/kinvolk/lokomotive/blob/v0.2.0/docs/configuration-reference/components/external-dns.md#attribute-reference
[✖] https://github.com/kinvolk/lokomotive/blob/v0.2.0/docs/configuration-reference/components/prometheus-operator.md
```

**Comment by iaguis** [comment]
> I assume that docs link pointing to `v0.2.0` will be resolved once the `v0.2.0` tag is created ?

Yes, you can test now by changing v0.2.0 with the commit that will be tagged:

* https://github.com/kinvolk/lokomotive/blob/b50d0f7a94add3191d125ca7bfd85d50fe125341/docs/configuration-reference/platforms/packet.md#attribute-reference
* https://github.com/kinvolk/lokomotive/blob/b50d0f7a94add3191d125ca7bfd85d50fe125341/docs/configuration-reference/components/external-dns.md#attribute-reference
* https://github.com/kinvolk/lokomotive/blob/b50d0f7a94add3191d125ca7bfd85d50fe125341/docs/configuration-reference/components/prometheus-operator.md

**Pull request merged**